vitrina/datasets/views.py

```diff
@@ -1205,14 +1206,28 @@ def get_history_objects(self):
         plan_history_objects = Version.objects.get_for_model(Plan).filter(
             object_id__in=list(dataset_plan_ids)
         )
+        dataset_distribution_history_objects = self._get_history_objects_for_model(DatasetDistribution)
+        attribution_history_objects = self._get_history_objects_for_model(DatasetAttribution)
+        relation_history_objects = self._get_history_objects_for_model(DatasetRelation)
 
         history_objects = (
             property_history_objects
             | model_history_objects
             | dataset_history_objects
             | plan_history_objects
+            | dataset_distribution_history_objects
+            | attribution_history_objects
+            | relation_history_objects
         )
         return history_objects.order_by("-revision__date_created")
+    
+    def _get_history_objects_for_model(self, model: TypingType[models.Model]) -> list[int]:
```

Comment:
This does not return a list, it returns a `QuerySet` of `Version` objects:
```suggestion
    def _get_history_objects_for_model(self, model: TypingType[models.Model]) -> QuerySet[Version]:
```